lib/index.js

```diff
@@ -367,8 +402,10 @@ class Service extends _adapterCommons.AdapterService {
     this.objectify(q, query);
 
     if (filters.$sort) {
-      Object.keys(filters.$sort).forEach(key => {
-        q.orderBy(key, filters.$sort[key] === 1 ? 'asc' : 'desc');
+      Object.keys(filters.$sort).forEach(item => {
```

Comment:
From the upstream.
https://github.com/feathersjs-ecosystem/feathers-objection/commit/dd854e177aec2e69a4e7d0a4676e4630b42e54b7